module/motion/HeadController/README.md

```diff
@@ -7,15 +7,19 @@ Controls the motion of the head
 
 ## Usage
 
+Takes a HeadCommand message:
 
+- If command requests smoothing (`command->smooth = true`), then the goal angle emitted to is smoothed using exponential filter, tunable using `smoothing_factor`
+- If command request is in world space (`command->goalRobotSpace = false`) the goal angle is converted from world space to robot space
 
-## Consumes
+The final goal angle is clamped based on the maximum/minium pitch and yaw of the head servos before being sent to servo controller
 
+## Consumes
 
+* `message::motion::HeadCommand` command containing desired angles, smoothing flag and reference frame information
 
 ## Emits
 
-
+*`message::behaviour::ServoCommands` instructs servo controller to move head to desired angle
 
```

Comment:
it emits a headcommand too (at the beginning), probably explain this.
emits a nusight graph, probably mention this
it also emits a registeraction, mention this (it registers the module to be allowed to move the servos, and it specifically only registers for the head and gives itself a high priority of 30 (more is higher priority). if it loses priority of the head, it disables the main reaction loop and if it gets back priority it enables the reaction again).